src/ciadmin/generate/worker_pools.py

```diff
@@ -201,109 +201,77 @@ def get_azure_provider_config(
             "wstServerURL", azure_config["wst_server_url"]
         )
     tags = config.get("tags", {})
-    # Temporary conditional language while we move pools over to the newer
-    # method that is in the else clause
-    # Refrence https://mozilla-hub.atlassian.net/browse/RELOPS-212
-    if config.get("old-style-worker-config"):
-        launch_configs = []
-        for location in locations:
-            for vmSize in vmSizes:
-                loc = location.replace("-", "")
-                ImageId = image.image_id(provider_id, location)
+
+    launch_configs = []
+    for location in locations:
+        for vmSize in vmSizes:
+            loc = location.replace("-", "")
+            DeploymentId = image.image_id(provider_id, "deployment_id")
+            try:
+                manifest = image.image_id(provider_id, "manifest")
+            except:
+                pass
```

Comment:
This `pass` will cause problems later because `manifest` (and `version`) will actually be undefined. So even referencing them will be a runtime error. Instead try doing `manifest = None`.